pages/common/xmlstarlet.md

```diff
@@ -0,0 +1,33 @@
+# xmlstarlet
+
+> A commandline XML/XSLT toolkit.
+> Note: You will likely need to know XPath: <https://developer.mozilla.org/en-US/docs/Web/XPath>.
+> More information: <https://xmlstar.sourceforge.net/docs.php>.
+
+- Format an XML document and print to stdout:
+
+`xmlstarlet format {{path/to/file.xml}}`
+
+- XML document can also be piped from stdin:
+
+`{{cat path/to/file.xml}} | xmlstarlet format`
+
+- Print all nodes that match a given XPath:
+
+`xmlstarlet select --template --copy-of {{xpath}} {{file.xml}}`
+
+- Insert an attribute to all matching nodes, and print to stdout (source file is unchanged):
+
+`xmlstarlet edit --insert {{xpath}} --type attr --name {{attribute_name}} --value {{attribute_value}} {{file.xml}}`
+
+- Rename some nodes, delete some nodes, and print to stdout (source file is unchanged):
+
+`xmlstarlet edit --rename {{xpath_to_rename}} --value {{new_name}} --delete {{xpath_to_delete}} {{file.xml}}`
+
+- Escape or unescape special XML characters in a given string:
+
+`xmlstarlet [un]escape {{string}}`
+
+- List a given directory as XML (omit argument to list current directory):
+
+`xmlstarlet ls [{{path/to/directory}}]`
```

Comment:
```suggestion
`xmlstarlet ls {{path/to/directory}}`
```

Reply:
Why?

Reply:
Are the brackets part of the command? Or are they supposed to indicate something else?

Reply:
> Are the brackets part of the command? Or are they supposed to indicate something else?

Taking a quick look at documentation, I don't think this is part of the command.